yb-voyager/cmd/exportData.go

```diff
@@ -138,14 +139,17 @@ func exportDataOffline() bool {
 	}
 	fmt.Printf("Initiating data export.\n")
 	utils.WaitGroup.Add(1)
-	go source.DB().ExportData(ctx, exportDir, finalTableList, quitChan, exportDataStart)
+	go source.DB().ExportData(ctx, exportDir, finalTableList, quitChan, exportDataStart, exportSuccessChan)
 	// Wait for the export data to start.
 	<-exportDataStart
 
 	updateFilePaths(&source, exportDir, tablesProgressMetadata)
 	if !disablePb {
 		utils.WaitGroup.Add(1)
-		exportDataStatus(ctx, tablesProgressMetadata, quitChan)
+		exportDataStatus(ctx, tablesProgressMetadata, quitChan, exportSuccessChan)
+	} else if source.DBType == MYSQL || source.DBType == ORACLE {
+		// TODO: Update tables metadata when PB are disabled.
+		<-exportSuccessChan
```

Comment:
Why is this required?

Reply:
Will change this notification channel to a buffered channel, we will need to change some other notification channels to buffered channels as well in a separate PR; we can probably accommodate this change in the PR for the interface-based implementation for the progress bars

Reply:
I would suggest to get this change, at least, for the `exportSuccessChan`. It will be very difficult to keep track of it once this PR gets merged.

Reply:
Ignore the previous comment--you have already taken care of it. Approved the PR.